src/app/personalised-homepage/page.js

```diff
@@ -99,11 +107,11 @@ function PersonalisedHomepage() {
                 </ItemRowContainer>
             </ItemContainer>
             <div className="min-w-[100vw]">
-                <SpotifyPlayer
+                {userPremium() ? <SpotifyPlayer
                     token={accessToken}
                     uris={[`spotify:${playerType}:${playTrack}`]}
                     play={playback}>
-                </SpotifyPlayer>
+                </SpotifyPlayer> : null}
```

Comment:
**Possible alternative:** 
We can also use a short-circuit evaluation `&&`, removing the need for the "else" statement.
```js
condition && action
```

If the left-hand side is true, the right side will be executed. Otherwise, nothing happens
```js
{userPremium() && (
  <SpotifyPlayer
    token={accessToken}
    uris={[`spotify:${playerType}:${playTrack}`]}
    play={playback}
  />
)}
```